### DISCUSSION TITLE
please , i want some feedbacks

### DISCUSSION THREAD

**Discussion by MLankaoui** [community/community#110966] [Programming Help]
### Select Topic Area

Show & Tell

### Body

i want you feedback on this project for a python beginner , still working on it

https://github.com/MLankaoui/PyChatBot/

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MLankaoui ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by lmdevv** [comment]
It looks okay, really beginner level, some minor feedback would be:
1. **Adding a "Getting Started" guide:** Help newbies by including easy steps on how to get your chatbot running, like how to download your code and how to interact with it. You can browse around github repos and see how they structure the readme.md file
2. **Skip the `.vscode` folder:** Depending on the project it might not be necessary, also not everyone uses VS Code, so you might want to leave this folder out of your project. You can add it to your `.gitignore`.

Overall great, keep it up and create more projects!
